OrbitService/CaptureServiceImpl.cpp

```diff
@@ -91,30 +91,48 @@ class GrpcCaptureEventSender final : public CaptureEventSender {
     CHECK(reader_writer_ != nullptr);
   }
 
+  ~GrpcCaptureEventSender() {
+    LOG("Total number of events sent: %lu", total_number_of_events_sent_);
+    LOG("Total number of bytes sent: %lu", total_number_of_bytes_sent_);
+    LOG("Average number of bytes per event: %.2f",
+        static_cast<float>(total_number_of_bytes_sent_) / total_number_of_events_sent_);
```

Comment:
I do not really know: is it ok to divide float to 0 in c++?

Reply:
Good catch, that would cause a divide by zero exception. Will fix.

Reply:
Done.